static/app/views/explore/contexts/logs/logsTableData.tsx

```diff
@@ -0,0 +1,44 @@
+import {createDefinedContext} from 'sentry/utils/performance/contexts/utils';
+import {
+  useLogsCursor,
+  useLogsSearch,
+} from 'sentry/views/explore/contexts/logs/logsPageParams';
+import {
+  useExploreLogsTable,
+  type UseExploreLogsTableResult,
+} from 'sentry/views/explore/logs/useLogsQuery';
+
+interface LogsTableData {
+  tableData: UseExploreLogsTableResult;
+}
+
+const [_LogsTableDataProvider, _useLogsTableData, LogsTableDataContext] =
+  createDefinedContext<LogsTableData>({
+    name: 'LogsTableDataContext',
+  });
```

Comment:
i would avoid adding contexts that make requests since we have a context that makes requests

Reply:
Already made the change, pushing.